sys/linux/dev_camx.txt.const

```diff
@@ -1,45 +1,45 @@
 # Code generated by syz-sysgen. DO NOT EDIT.
-arches = arm64
+arches = arm, arm64
 AT_FDCWD = ???
```

Comment:
Not directly related to this change, but for this one and O_RDWR you need to <uapi/linux/fcntl.h>.